utxonursery.go

```diff
@@ -1002,16 +820,30 @@ func (u *utxoNursery) registerSweepConf(finalTx *wire.MsgTx,
 // to mark any mature channels as fully closed in channeldb.
 // NOTE(conner): this method MUST be called as a go routine.
 func (u *utxoNursery) waitForSweepConf(classHeight uint32,
-	kgtnOutputs []kidOutput, confChan *chainntnfs.ConfirmationEvent) {
+	output *kidOutput, resultChan chan sweep.Result) {
 
 	defer u.wg.Done()
 
 	select {
-	case _, ok := <-confChan.Confirmed:
+	case result, ok := <-resultChan:
 		if !ok {
-			utxnLog.Errorf("Notification chan closed, can't"+
-				" advance %v graduating outputs",
-				len(kgtnOutputs))
+			utxnLog.Errorf("Notification chan closed, can't" +
+				" advance graduating output")
+			return
+		}
+
+		// In case of a remote spend, still graduate the output. There
+		// is no way to sweep it anymore.
+		if result.Err == sweep.ErrRemoteSpend {
+			utxnLog.Infof("Output %v was spend by remote party",
```

Comment:
warn?

Reply:
I think this is still happy flow, therefore `Info`.